docs/cloud-native-security/get-started-with-kspm.asciidoc

```diff
@@ -0,0 +1,189 @@
+* <<kspm-use-iam-arn,Use an IAM role ARN>>
+
+Regardless of which option you use, you'll need to grant the following permissions:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "ecr:GetRegistryPolicy",
+                "eks:ListTagsForResource",
+                "elasticloadbalancing:DescribeTags",
+                "ecr-public:DescribeRegistries",
+                "ecr:DescribeRegistry",
+                "elasticloadbalancing:DescribeLoadBalancerPolicyTypes",
+                "ecr:ListImages",
+                "ecr-public:GetRepositoryPolicy",
+                "elasticloadbalancing:DescribeLoadBalancerAttributes",
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "ecr-public:DescribeRepositories",
+                "eks:DescribeNodegroup",
+                "ecr:DescribeImages",
+                "elasticloadbalancing:DescribeLoadBalancerPolicies",
+                "ecr:DescribeRepositories",
+                "eks:DescribeCluster",
+                "eks:ListClusters",
+                "elasticloadbalancing:DescribeInstanceHealth",
+                "ecr:GetRepositoryPolicy"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+[discrete]
+[[kspm-use-keys-directly]]
+===== Option 1 - Use access keys directly
+Access keys are long-term credentials for an IAM user or the AWS account root user. To use access keys as credentials, you need to provide the `Access key ID` and the `Secret Access Key`.
+
+For more details refer to AWS's documentation for https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html[Access Keys and Secret Access Keys].
+
+IMPORTANT: You must select "Programmatic access" when creating the IAM user.
+
+[discrete]
+[[kspm-use-temp-credentials]]
+===== Option 2 - Use temporary security credentials
+Temporary security credentials can be configured in AWS to last for a specified period of time. They consist of an access key ID, a secret access key, and a security token, which is typically found using `GetSessionToken`.
+
+Because temporary security credentials are short term, once they expire you will need to generate new ones and manually update the integration's configuration to continue collecting cloud posture data. Update the credentials before they expire to avoid data loss.
+
+NOTE: IAM users with multi-factor authentication (MFA) enabled need to submit an MFA code when calling `GetSessionToken`. For more details refer to AWS's documentation for https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[Temporary Security Credentials].
+
+You can use the AWS CLI to generate temporary credentials. For example, you could use the following command if you have MFA enabled: `sts get-session-token --serial-number arn:aws:iam::1234:mfa/your-email@example.com --duration-seconds 129600 --token-code 123456`
```

Comment:
Same comment about wrapping this in a console block. At the very least, it should prob be on a new line.